Mathlib/NumberTheory/Padics/PadicVal.lean

```diff
@@ -448,6 +481,21 @@ theorem sum_pos_of_pos {n : ℕ} {F : ℕ → ℚ} (hF : ∀ i, i < n → 0 < pa
         exact hF _ (lt_trans hi (lt_add_one _))
 #align padic_val_rat.sum_pos_of_pos padicValRat.sum_pos_of_pos
 
+/-- If the p-adic valuation of a finite set of positive rationals is greater than a given rational
+number, then the p-adic valuation of their sum is also greater than the same rational number. -/
+theorem finset_gen_sum_lt_of_lt {p j : ℕ} [hp : Fact (Nat.Prime p)]
```

Comment:
what is `gen`?

Reply:
Renamed to `sum_gt_of_gt` to conform with`sum_pos_of_pos` right above it.